microsoft-edge/webview2/concepts/distribution.md

```diff
@@ -52,6 +52,8 @@ The Evergreen Runtime is preinstalled onto all Windows 11 devices as a part of t
 
 In the _Fixed Version_ distribution mode, you download a specific version of the WebView2 Runtime and package it together with your WebView2 app in your app package.  The WebView2 Runtime that you package with your app is used only by your WebView2 app, not by any other apps on the client's machine.
 
+The Fixed Version runtime distribution mode is also called a "self-contained" distribution model; the redistributable bits are included with your app.  This approach is similar to .NET's "Self-Contained deployment option".
```

Comment:
We don't have special support for .NET self-contained apps to use WebView2. Also, .NET self-contained apps are similar at a high level but different otherwise in practical usage. I don't think we should add this paragraph.